leftwm-core/src/handlers/command_handler.rs

```diff
@@ -143,6 +143,38 @@ fn toggle_state(state: &mut State, window_state: WindowState) -> Option<bool> {
     let window = state.focus_manager.window(&state.windows)?;
     let handle = window.handle;
     let toggle_to = !window.has_state(&window_state);
+    let tag_id = state.focus_manager.tag(0)?;
+
+    let for_active_workspace = |x: &Window| -> bool { x.tag == Some(tag_id) && x.is_managed() };
+
+    if window_state == WindowState::Fullscreen {
+        //Going fullscreen, so we save the window order
+        //or else, we restore it!
+        if toggle_to {
+            let mut to_reorder = helpers::vec_extract(&mut state.windows, for_active_workspace);
+            let handles = to_reorder
+                .iter()
+                .map(|each_handle| each_handle.handle)
```

Comment:
Would it be better to call the input to the closure wither `w` or `window` since `to_reorder` contains windows and not handles when `map` is called?

Reply:
I assume storing only the handlers was intended to be the cheaper operation, no?